deployment/deploy.js

```diff
@@ -14,14 +14,41 @@
  *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
  *  PERFORMANCE OF THIS SOFTWARE.
  */
-const Deployer = require('forgae-lib').Deployer;
+const { Universal, MemoryAccount, Node, Crypto } = require('@aeternity/aepp-sdk');
+const contractUtils = require('../utils/contract-utils');
 
-const deploy = async (network, privateKey, compiler) => {
-    let deployer = new Deployer(network, privateKey, compiler)
+const NETWORKS = require('../config/network.json');
+const DEFAULT_NETWORK_NAME = 'local';
 
-    await deployer.deploy("./contracts/token-registry.aes")
+const EXAMPLE_CONTRACT_SOURCE = './contracts/ExampleContract.aes';
```

Comment:
```suggestion
const EXAMPLE_CONTRACT_SOURCE = './contracts/token-registry.aes';
```

Reply:
Not sure if you are using `aeproject deploy`.

Reply:
thanks, just an oversight